docs/README_overriders.md

```diff
@@ -0,0 +1,13 @@
+# Developer documentation for MakeOverride Command
```

Comment:
A améliorer, je pense que c'est mieux de faire une document claire ici

Reply:
c'est noté aussi.

Reply:
Ce qui serait top, c'est que quelqu'un essai de faire un overrider et fasse la doc en même temps..